### PR TITLE
Update HWT links with the new domain

### PR DESCRIPTION
**Pull request by lheagy**
update HWT url to http://hohmannwannamakertrust.org/

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ highlight_code = false
 navlinks = [
     {name="Registration", url="/#registration"},
     {name="Call for Abstracts", url="/#abstracts-call"},
-    {name="HWT homepage", url="https://www.gwhohmann.org/", external=true},
+    {name="HWT homepage", url="http://hohmannwannamakertrust.org/", external=true},
```
Comment (santisoler) [inline config.toml]:
```suggestion
    {name="HWT homepage", url="https://hohmannwannamakertrust.org/", external=true},
```

File: content/_index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ sponsors = [
 
 # Organizers
 organizers = [
-    {image="hwt.png", alt_text="Hohmann-Wannamaker Trust", url="https://www.gwhohmann.org/"},
+    {image="hwt.png", alt_text="Hohmann-Wannamaker Trust", url="http://hohmannwannamakertrust.org/"},
```
Comment (santisoler) [inline content/_index.md]:
```suggestion
    {image="hwt.png", alt_text="Hohmann-Wannamaker Trust", url="https://hohmannwannamakertrust.org/"},
```

File: content/_index.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ abstracts_call = """
 """
 
 hwt_about = """
-The 3DEM symposium is organized by the [Hohmann-Wannamaker Trust](https://www.gwhohmann.org/).
+The 3DEM symposium is organized by the [Hohmann-Wannamaker Trust](http://hohmannwannamakertrust.org/).
```
Comment (santisoler) [inline content/_index.md]:
```suggestion
The 3DEM symposium is organized by the [Hohmann-Wannamaker Trust](https://hohmannwannamakertrust.org/).
```

**Review by santisoler — APPROVED** [review]
The new website looks great! Thanks for updating the links. I've just added the `s` in `https`, since they do have TLS configured.

**Comment by lheagy** [comment]
:+1: Thanks Santi!

**Pull request merged**